### ISSUE TITLE
Better error handling

### ISSUE THREAD

**Issue by andreasonny83** [lukehaas/runjs#2]
The error handling is quite awkward because it doesn't provide much help in debugging the problems.

Take the following code, for example

```
const a () => new Promise(res => res('test'));

const test = async () => {
  const x = await a();
  
  return x;
};

test();
```

All I can see on the right side is a `SyntaxError: Unexpected identifier`

The error is located on the first line of my code:

```
const a () => new Promise(res => res('test'));
```

The arrow function is missing the `=` sign after `const a`.

It would be really useful if the error message could mention the line of code that was the trigger of the error. Ideally, it would also output the explanation of the error like:

```
Uncaught SyntaxError: Missing initializer in const declaration
```

Thanks

**Comment by lukehaas** [comment]
Yes, certainly something that could be improved in future versions 👍

**Comment by gianpesto** [comment]
I was about to write the same, one solution could be to have the Error-message and the Error-source being highlighted in the same color.

**Comment by comatory** [comment]
Yes please this would be very useful, perhaps highlight the line or at least print out line & column number

**Comment by lukehaas** [comment]
This is now resolved in version 1.2.1

The above code will now result in the following error:


![screenshot 2019-01-27 at 16 37 55](https://user-images.githubusercontent.com/741344/51803891-f0d99f00-2251-11e9-8fa3-8637fd0f0f87.png)

**Comment by andreasonny83** [comment]
That's great news. Closing this issue then.
